draft-ietf-quic-transport.md

```diff
@@ -208,1721 +217,1890 @@ x (*) ...
 : Indicates that x is variable-length
```

Comment:
This section on encoding and pictures could be moved much later in the document.  We don't really use this until we start describing the wire format, and if we had a section on notation conventions before we got into packet formats and the like, we could put this with the note on network byte order and varint encoding.